kubernetes/resource_kubernetes_stateful_set_test.go

```diff
@@ -148,6 +148,35 @@ func TestAccKubernetesStatefulSet_update_replicas(t *testing.T) {
 		},
 	})
 }
+
+func TestAccKubernetesStatefulSet_update_replicas_to_zero(t *testing.T) {
+	var conf api.StatefulSet
+	name := fmt.Sprintf("tf-acc-test-%s", acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum))
+	resource.Test(t, resource.TestCase{
+		PreCheck:      func() { testAccPreCheck(t) },
+		IDRefreshName: statefulSetTestResourceName,
+		Providers:     testAccProviders,
+		CheckDestroy:  testAccCheckKubernetesStatefulSetDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccKubernetesStatefulSetConfigBasic(name),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					testAccCheckKubernetesStatefulSetExists(statefulSetTestResourceName, &conf),
+					testAccKubernetesStatefulSetChecksBasic(name),
+				),
+			},
+			{
+				Config: testAccKubernetesStatefulSetConfigUpdateReplicas(name),
```

Comment:
Doesn't the config returned by that function set replicas to 5?

https://github.com/terraform-providers/terraform-provider-kubernetes/blob/9c3d1a4942a0c0a8a1285446a0b016260ee0dd92/kubernetes/resource_kubernetes_stateful_set_test.go#L645

Reply:
It does, thank you - I have amended the commit.